products/ovirt.md

```diff
@@ -0,0 +1,61 @@
+---
+title: oVirt
+category: os
+permalink: /ovirt
+changelogTemplate: "https://www.ovirt.org/release/__LATEST__"
+releaseDateColumn: true
+
+auto:
+-   git: https://github.com/oVirt/ovirt-engine
+    regex: ^v(?<major>\d+)_(?<minor>\d+)_(?<patch>\d{1,3})_?(?<tiny>\d+)?$
+
+releases:
```

Comment:
```suggestion
# eol(x) = releaseDate(x+1)
releases:
```

For future maintainers (based on the description).

All releases having `eol: false` must also be updated accordingly.

Reply:
Applied

I also updated the `eol: true` to match the release date of the x+1